### PR TITLE
Add documentation for manual testing

### PR DESCRIPTION
**Pull request by nacho-undefinedlabs**

### CODE REVIEW

File: docs/ios-manual-testing.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+id: ios-manual-testing
+title: Scope iOS Agent Manual testing
+sidebar_label: Manual testing
+---
+
+Scope allows to do manual test runs that automatically connect to your Scope instance. You can run your application from Xcode in your simulator or device, but you can also distribute a build to your testers with the functionality ready.
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
The Scope iOS agent allows you to perform manual tests on your application that will be recorded for later troubleshooting in Scope. You can either run your application from Xcode in your simulator or device, or distribute a build to your testers with the functionality built-in.
```

File: docs/ios-manual-testing.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+The results will appear in your local development scratchpad in Scope
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
The results will appear in your **Local Development scratchpad** in Scope
```

File: docs/ios-manual-testing.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+```
+binary "https://releases.undefinedlabs.com/scope/agents/ios/ScopeAgent.json"
+```
+
+In your application or framework targets, add `ScopeAgent.framework` located in `Carthage/Build/iOS` to the
+*Linked frameworks and Libraries* in General target settings or to the *Link Binaries With Libraries* build phase. 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+Then you must add the following entries in your application Info.plist
```
Comment (fermayo) [inline docs/ios-manual-testing.md]: This step depends on whether the user wants to hardcode the values for distribution or not. I would put first the instructions of the env vars for local Xcode runs, and later, as an optional step, update the `Info.plist` if the user wants to distribute the application with manual testing activated.

File: docs/ios-manual-testing.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+| Key                        | Value                         |
+| -------------------------- | ----------------------------- |
+| `SCOPE_XCODE_APIKEY`       | `$(SCOPE_XCODE_APIKEY)`       |
+| `SCOPE_XCODE_API_ENDPOINT` | `$(SCOPE_XCODE_API_ENDPOINT)` |
+
+
+## Usage
+
+After the application launches, a Scope logo will appear in the screen over your application elements. When you want to start a test you must simple press, and it will allow to start a new test.
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
After the application launches, a floating Scope logo will appear in the screen over your application elements. When you want to start a test, click on the logo and provide the test with a name.
```

File: docs/ios-manual-testing.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+ While the test is running the Scope logo will be glowing in red, if you click the logo again it will allow to stop the recording and save the state of the test.
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
While the test is being recorded, the Scope logo will be glowing in red. To finish the test and stop the recording, click on the logo, and choose whether the test passed or failed. 
```

File: docs/ios-manual-testing.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+| `SCOPE_XCODE_API_ENDPOINT` | `$(SCOPE_XCODE_API_ENDPOINT)` |
+
+
+## Usage
+
+After the application launches, a Scope logo will appear in the screen over your application elements. When you want to start a test you must simple press, and it will allow to start a new test.
+
+ While the test is running the Scope logo will be glowing in red, if you click the logo again it will allow to stop the recording and save the state of the test. 
+
+After saving, the test will appear in Scope.
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
Manual tests will appear in your **Local Development scratchpad** section.
```

File: docs/ios-manual-testing.md (hunk 3 above)
Comment (nacho-undefinedlabs) [reply docs/ios-manual-testing.md]: Changed

File: docs/ios-manual-testing.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+```
+binary "https://releases.undefinedlabs.com/scope/agents/ios/ScopeAgent.json"
+```
+
+In your application or framework targets, add `ScopeAgent.framework` located in `Carthage/Build/iOS` to the
+*Linked frameworks and Libraries* in General target settings or to the *Link Binaries With Libraries* build phase. 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+ If you want to run your application from Xcode in your simulator or device, the following environment variables must be set in your **Run target** ([instructions](https://help.apple.com/xcode/mac/10.1/index.html?localePath=en.lproj#/dev3ec8a1cb4)):
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
If you want to run your application from Xcode in your simulator or device, the following environment variables must be set in your **Run target** ([instructions](https://help.apple.com/xcode/mac/10.1/index.html?localePath=en.lproj#/dev3ec8a1cb4)):
```

File: docs/ios-manual-testing.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+| Key                        | Value                         |
+| -------------------------- | ----------------------------- |
+| `SCOPE_XCODE_APIKEY`       | `$(SCOPE_XCODE_APIKEY)`       |
+| `SCOPE_XCODE_API_ENDPOINT` | `$(SCOPE_XCODE_API_ENDPOINT)` |
+
+Alternatively, if you want to distribute the instrumented application to you QA team, then you must add the following entries in your application Info.plist:
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
Alternatively, if you want to distribute the instrumented application (e.g. to the QA team), then you must add the following entries in your application's `Info.plist`:
```

File: docs/ios-manual-testing.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+| Key                  | Type     | Value                         |
+| -------------------- | -------- | ----------------------------- |
+| `SCOPE_APIKEY`       | `String` | `< your Scope APIKEY >`       |
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
| `SCOPE_APIKEY`       | `String` | `<Your Scope API key>`        |
```

File: docs/ios-manual-testing.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+| `SCOPE_API_ENDPOINT` | `String` | `< your Scope API_ENDPOINT >` |
```
Comment (fermayo) [inline docs/ios-manual-testing.md]:
```suggestion
| `SCOPE_API_ENDPOINT` | `String` | `<Your Scope API endpoint>`   |
```

**Pull request merged**